src/pal/src/configure.cmake

```diff
@@ -73,8 +73,33 @@ check_function_exists(directio HAVE_DIRECTIO)
 check_function_exists(semget HAS_SYSV_SEMAPHORES)
 check_function_exists(pthread_mutex_init HAS_PTHREAD_MUTEXES)
 check_function_exists(ttrace HAVE_TTRACE)
+if(CMAKE_SYSTEM_NAME STREQUAL Linux)
```

Comment:
I wonder if we need to specify the libraries at all. We are not trying to execute the code, just testing if it compiles or not and unless I am mistaken, the missing library would only be detected at runtime.